protocols/indexer/contracts/Indexer.sol

```diff
@@ -131,9 +131,9 @@ contract Indexer is IIndexer, Ownable {
 
     // Ensure both of the tokens are not blacklisted.
     require(!blacklist[_makerToken] && !blacklist[_takerToken],
-      "INDEX_IS_BLACKLISTED");
+      "TOKEN_IS_BLACKLISTED");
```

Comment:
slightly vague message given that TOKEN could be referring to _makerToken or _takerToken.
could just be PAIR_IS_BLACKLISTED

Reply:
It could be either, in which case, we should consider two separate `require` statements. Thoughts?

Reply:
given someone can lookup the token pair freely once they receive the message, since blacklist is public, i don't think having two `require` statements is to helpful.

Reply:
Okay let's go with it.